src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApi.java

```diff
@@ -354,9 +385,16 @@ public CaseDetails submitEventForAttachScannedDocs(
                 e
             );
         } catch (FeignException e) {
+            removeFromIdamCache(e, jurisdiction);
             throw new CcdCallException(
                 String.format("Could not attach documents for case ref: %s Error: %s", caseRef, e.status()), e
             );
         }
     }
+
+    private void removeFromIdamCache(FeignException ex, String jurisdiction) {
```

Comment:
I got mislead by the name of this method. How about `removeFromIdamCacheIfAuthProblem` or something like this, so that we know the token is removed conditionally here?

Reply:
org.springframework.http.HttpStatus.FORBIDDEN.value(), org.springframework.http.HttpStatus.UNAUTHORIZED.value()

Reply:
The first parameter should be status itself rather than the whole exception object.

Reply:
change the method